editors/code/package.json

```
@@ -278,6 +278,21 @@
                     "type": "number",
                     "default": 20,
                     "description": "Maximum length for inlay hints"
+                },
+                "rust-analyzer.cargoFeatures.noDefaultFeatures": {
+                    "type": "boolean",
+                    "default": false,
+                    "description": "Do not activate the `default` feature"
+                },
+                "rust-analyzer.cargoFeatures.allFeatures": {
+                    "type": "boolean",
+                    "default": false,
```

```suggestion
                    "default": true,
```

I think that features a *usually* additive, so by default `allFeatures` make slightly more sense for our use-case.